### PR TITLE
add new types for credentialStatus

### PR DESCRIPTION
**Pull request by ilya-korotya**

### CODE REVIEW

File: core/jsonld/iden3proofs.jsonld [hunk 1]
```diff
@@ -239,6 +239,46 @@
...
       }
+    },
+    "Iden3commRevocationStatusV1.0": {
+      "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/iden3credential-v2.json-ld#Iden3commRevocationStatusV1.0",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "iden3-comm-revocation-statusV1.0-vocab": "https://github.com/iden3/claim-schema-vocab/blob/main/proofs/Iden3commRevocationStatusV1.0.md#",
```
Comment (vmidyllic) [inline core/jsonld/iden3proofs.jsonld:250]: it must link to file in core/vocab

File: core/jsonld/iden3proofs.jsonld [hunk 2]
```diff
@@ -239,6 +239,46 @@
...
+      }
+    },
+    "Iden3OnchainSparseMerkleTreeProof2023": {
+      "@id": "https://raw.githubusercontent.com/iden3/claim-schema-vocab/main/schemas/json-ld/iden3credential-v2.json-ld#Iden3OnchainSparseMerkleTreeProof2023",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "iden3-onchain-sparse-merkle-tree-proof-2023-vocab": "https://github.com/iden3/claim-schema-vocab/blob/main/proofs/Iden3OnchainSparseMerkleTreeProof2023.md#",
```
Comment (vmidyllic) [inline core/jsonld/iden3proofs.jsonld:270]: it must link to core/vocab

File: core/jsonld/iden3proofs.jsonld (hunk 1 above)
Comment (ilya-korotya) [reply core/jsonld/iden3proofs.jsonld]: done

File: core/jsonld/iden3proofs.jsonld (hunk 2 above)
Comment (ilya-korotya) [reply core/jsonld/iden3proofs.jsonld]: done

**Review by vmidyllic — APPROVED** [review]

**Pull request merged**